diff --git a/node_modules/@expo/image-utils/build/Cache.js b/node_modules/@expo/image-utils/build/Cache.js
index 85adb45..f5cecff 100644
--- a/node_modules/@expo/image-utils/build/Cache.js
+++ b/node_modules/@expo/image-utils/build/Cache.js
@@ -12,7 +12,7 @@ exports.clearUnusedCachesAsync = clearUnusedCachesAsync;
 const crypto_1 = __importDefault(require("crypto"));
 const fs_1 = __importDefault(require("fs"));
 const path_1 = require("path");
-const CACHE_LOCATION = '.expo/web/cache/production/images';
+const CACHE_LOCATION = '/tmp/expo-image-cache';
 const cacheKeys = {};
 // Calculate SHA256 Checksum value of a file based on its contents
 function calculateHash(filePath) {
@@ -36,7 +36,7 @@ async function createCacheKeyWithDirectoryAsync(projectRoot, type, icon) {
     return cacheKey;
 }
 async function ensureCacheDirectory(projectRoot, type, cacheKey) {
-    const cacheFolder = (0, path_1.join)(projectRoot, CACHE_LOCATION, type, cacheKey);
+    const cacheFolder = (0, path_1.join)(CACHE_LOCATION, type, cacheKey);
     await fs_1.default.promises.mkdir(cacheFolder, { recursive: true });
     return cacheFolder;
 }
@@ -58,7 +58,7 @@ async function cacheImageAsync(fileName, buffer, cacheKey) {
 }
 async function clearUnusedCachesAsync(projectRoot, type) {
     // Clean up any old caches
-    const cacheFolder = (0, path_1.join)(projectRoot, CACHE_LOCATION, type);
+    const cacheFolder = (0, path_1.join)(CACHE_LOCATION, type);
     await fs_1.default.promises.mkdir(cacheFolder, { recursive: true });
     const currentCaches = await fs_1.default.promises.readdir(cacheFolder);
     if (!Array.isArray(currentCaches)) {
